plugins/inputs/x509_cert/x509_cert.go

```diff
@@ -280,7 +280,6 @@ func init() {
 	inputs.Add("x509_cert", func() telegraf.Input {
 		return &X509Cert{
 			Sources: []string{},
-			Timeout: internal.Duration{Duration: 5},
```

Comment:
This is not the correct fix. You should instead initialize the duration correctly as `5*time.Second`.